src/SFA.DAS.AANHub.Application/Profiles/Queries/GetProfiles/GetProfilesQuery.cs

```diff
@@ -0,0 +1,11 @@
+﻿using MediatR;
+using SFA.DAS.AANHub.Application.Mediatr.Responses;
+
+namespace SFA.DAS.AANHub.Application.Profiles.Queries.GetProfiles
+{
+    public class GetProfilesQuery : IRequest<ValidatedResponse<List<ProfileModel>>>
```

Comment:
Rename this and all other respective files including tests, to use `GetProfilesByUserType`